articles/google-play-in-app-reviews/index.md

```diff
@@ -0,0 +1,250 @@
+
+1. How you ask for reviews matters. When using this API, you should not ask predictive or opinion questions before or after the review sheet dialogs show the app in your application. These questions include:
+
+    -  Would you rate this application five stars?
+    -  Do you like this app?
+
+2. This API also advises you only to show dialog once a user has enough experience with your application. It's not a good practice to ask a user to write a review before the app delivers value to that user. Chances are, the user will get annoyed and skip the review. And if they don't skip it, they will give negative reviews.
+
+3. In order to minimize API usage and avoid annoying users, it would be best if you did not excessively ask users to write reviews.
+
+### Implementation
+Google describes this API as light, and indeed it is. It is simple to integrate within your application. To begin, identify an activity where you want to implement the review flow. Once implemented, the API will handle the review flow for you.
+
+#### Requirements
+- An Android phone running android 5.0 (API 21) or higher. The device should have Google Play installed and connected with a Gmail account.
+- Google Play Core Library, version 1.8.0 or higher.
+
+#### Integration
+
+##### Step 1: Adding dependency
+Add Google Core Library to your `app.gradle` file.
+
+Sync to download the library to have access to the necessary classes to initiate review flow.
+
+Always ensure that you are using the latest library version. Check [here](https://developer.android.com/reference/com/google/android/play/core/release-notes) for newly added versions.
+
+##### Step 2: Creating an instance of a `ReviewManager`
+The `ReviewManager` provides the required functions that trigger the review flow.
+They include;
+- `requestReviewFlow()` — fetches application information from the Play store.
+- `launchReviewFlow()` — initiates the review flow.
+
+Declare `ReviewManager` right above `onCreate()`.
+
+To create an instance of `ReviewManager`,
+- Declare `ReviewManager` right above `onCreate()`.
+- Use `ReviewManagerFactory` calling the `create` function, and pass your application context to it.
+
+```java
+reviewManager = ReviewManagerFactory.create(getApplicationContext());
+```
+
+##### Step 3: Requesting review info
+`requestReviewFlow()` communicates with the Google Play store remotely to get the information that references your application. Declare `ReviewInfo`.
+
+```java
+ReviewInfo reviewlnfo;
+```
+
+`ReviewInfo` holds this information, which will be used to trigger the review flow process to the end user.
+
+```js
+Task<ReviewInfo> manager = reviewManager.requestReviewFlow();
+        manager.addOnCompleteListener(task -> {
+            if (task.isSuccessful()) {
+                reviewlnfo = task.getResult();
```

Comment:
rename the variable `reviewlnfo` to `reviewInfo` and replace all the instances. Grammar is also vital in code.